MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -49,7 +49,7 @@ class RouteManeuverViewController: UIViewController {
                 return
             }
             
-            if components.count > 1 {
+            if components.count == 2 || (components.count == 3 && ["North", "South", "East", "West", "Nord", "Sud", "Est", "Ouest"].contains(components[2])) {
```

Comment:
This array contains the cardinal directions in English (for the United States and most of Canada) and French (for Québec). These appear to be the only regions that put cardinal directions in the [`destination:ref`](https://taginfo.openstreetmap.org/keys/destination:ref) tag in OpenStreetMap, and besides, Shields.plist only covers the U.S. at the moment (#334). I refrained from making these strings localizable, because we need to continue to match cardinal directions in road codes regardless of whether the user is using the country’s official language.